lib/fizzy/execute.cpp

```diff
@@ -634,12 +625,14 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, const Value* args,
                 goto trap;
 
             // check actual type against expected type
-            const auto& actual_type = called_func->type;
```

Comment:
Why is this not working anymore when the `called_func` declaration above hasn't changed?